doc/conf.py

```diff
@@ -182,17 +183,21 @@
      'Miscellaneous'),
 ]
 
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None),
+    'mne': ('http://mne-tools.github.io/stable/', None),
+    'numpy': ('https://www.numpy.org/devdocs', None),
+    'scipy': ('https://scipy.github.io/devdocs', None),
+    'matplotlib': ('https://matplotlib.org', None),
+    'nilearn': ('http://nilearn.github.io', None),
+}
 
 sphinx_gallery_conf = {
     'examples_dirs': '../examples',
     'gallery_dirs': 'auto_examples',
     'filename_pattern': '^((?!sgskip).)*$',
     'backreferences_dir': 'generated',
-    'reference_url': {
-        'mne': 'http://mne-tools.github.io/stable/',
-        'numpy': 'http://docs.scipy.org/doc/numpy-1.9.1',
-        'scipy': 'http://docs.scipy.org/doc/scipy-0.17.0/reference'
-    },
```

Comment:
why did you do this? Examples are cross-referenced to the documentation of other libraries using this, no?

Reply:
yes, but I did it properly now using intersphinx

Reply:
see line 186 above